docs/multi-cloud-deployment-guide.md

```diff
@@ -0,0 +1,402 @@
+# Multi-Cloud DocumentDB Deployment Guide
+
+This guide provides step-by-step instructions for setting up a multi-cloud
+deployment of DocumentDB using KubeFleet to manage clusters across
+clouds. This setup enables high availability and disaster recovery.
+We assume the use of an AKS cluster and an on-prem Kubernetes cluster
+that have network access to one another.
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Architecture Overview](#architecture-overview)
+3. [Setting Up the Hub Cluster](#setting-up-the-hub-cluster)
+4. [Adding Clusters to Fleet](#adding-clusters-to-fleet)
+5. [Installing Operators and Dependencies](#installing-operators-and-dependencies)
+6. [Deploying DocumentDB Operator to Fleet](#deploying-documentdb-operator-to-fleet)
+7. [Setting Up Replication](#setting-up-replication)
+8. [Testing and Verification](#testing-and-verification)
+9. [Failover Procedures](#failover-procedures)
+
+## Prerequisites
+
+- Azure account
+- Azure CLI installed and configured with appropriate permissions
+- kubectl installed
+- helm installed
+- Git client
+- Two kubernetes clusters that are network connected to each other
```

Comment:
say to use express route or https://learn.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-about-vpngateways or similar